libs/cvr-fixture-generator/src/cli/generate/main.ts

```diff
@@ -58,6 +52,8 @@ interface IO {
 
 /**
  * Command line interface for generating a cast vote record file.
+ *
+ * TODO: Make more full use of export functions in libs/backend to avoid duplicating logic.
```

Comment:
I'm pretty excited about this idea! My thinking:

- Replace the CVR-generating function in this lib with a function that generates `ResultSheet` objects
- Prepare a mock USB drive and mock store and pass them and the above `ResultSheet` objects to `exportCastVoteRecordsToUsbDrive` from `libs/backend`, to have that function do all the heavy lifting of generating a CVR fixture

Then, we wouldn't have to worry about keeping `libs/backend` and `libs/cvr-fixture-generator` in sync.

I'd tackle this now, but I want to first finish removing old code, and this is a bit of a tangent.

Reply:
That would be wonderful. As such it's like we are maintaining a separate CVR format entirely.

Reply:
My only concern is that _ideally_, we'd like to be able to auto-generate CVRs in `libs/fixtures`. A little bit ago I created #3905.

Introducing more dependencies could make that harder.

But perhaps the solution is to allow those dependencies and _not_ merge the two libraries, but rather use scripting to call one library from the other.

Reply:
Ah yes thanks for pointing that out! I like the idea of using scripts to call from one library to another, instead of a hard dependency